shared/css/conversation.css

```diff
@@ -364,11 +364,59 @@ html[dir="rtl"] .settings-menu.dropdown-menu {
 /*
  * For any audio-only streams, we want to display our own background
  */
+.avatar-container {
+  width:  100%;
+  height: 100%;
+  background-color: #d8d8d8;
+  overflow: hidden;
+}
+
+.media-wrapper > .remote > .avatar-container {
+  background-color: #00a9dc;
+}
+
+.avatar-volume {
+  border-radius: 50%;
+  background-color: #e8e8e8;
+  position: relative;
+  /* Default size */
+  height: 146px;
+  width: 146px;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+}
+
+/* Set avatar sizes by media-wrapper selectors */
+.media-wrapper > .remote .avatar-volume {
+  background-color: #78d2ee;
+}
+
+.media-wrapper .local .avatar-volume {
+  background-color: #e8e8e8;
+}
+
+.media-wrapper:not(.receiving-screen-share) > .remote .avatar-volume,
+.media-wrapper:not(.showing-remote-streams) > .focus-stream > .local
+{
+  height: 146px;
+  width: 146px;
```

Comment:
You've already specified these are the default size above, so I don't see you'd need to repeat them.